tests/test-slirp4netns-configure.sh

```diff
@@ -8,7 +8,7 @@ child=$!
 
 wait_for_network_namespace $child
 
-slirp4netns -c $child tun11 &
+slirp4netns --enable-seccomp -c $child tun11 &
```

Comment:
Hmm.. This was supposed to fail but it didn't. I'll have to do some more testing..